.goreleaser.yml

```diff
@@ -0,0 +1,97 @@
+project_name: bosh_exporter
+
+builds:
+  - id: binary
+    dir: .
+    main: ./main.go
+    binary: bosh_exporter
+    env:
+      - CGO_ENABLED=0
+    ldflags:
+      - -s
+      - -w
+      - -X github.com/prometheus/common/version.Version={{.Version}}
+      - -X github.com/prometheus/common/version.Revision={{.FullCommit}}
+      - -X github.com/prometheus/common/version.Branch={{.Branch}}
+      - -X github.com/prometheus/common/version.BuildUser={{.Env.USER}}
+      - -X github.com/prometheus/common/version.BuildDate={{.Date}}
+      - -X code.cloudfoundry.org/cli/version/binaryVersion={{.Version}}
+    goos:
+      - linux
+      - darwin
+      - windows
+    goarch:
+      - amd64
+      - 386
+      - arm
+      - arm64
+      - ppc64
+      - ppc64le
+    goarm: [5,6,7]
+    ignore:
+      - goos: darwin
+        goarch: arm
+      - goos: darwin
+        goarch: arm64
+      - goos: darwin
+        goarch: ppc64
+      - goos: darwin
+        goarch: ppc64le
+      - goos: darwin
+        goarch: 386
+      - goos: windows
+        goarch: arm
+      - goos: windows
+        goarch: arm64
+      - goos: windows
+        goarch: ppc64
+      - goos: windows
+        goarch: ppc64le
+
+archives:
+  - id: archives
+    builds: [binary]
+    format: "tar.gz"
+    name_template: '{{ .ProjectName }}_{{ .Version }}.{{ .Os }}-{{ .Arch }}{{ with .Arm }}v{{ . }}{{ end }}'
+    wrap_in_directory: true
+    files:
+      - LICENSE
+      - NOTICE
+      - README*
+      - CHANGELOG*
+      - docs/*
+    format_overrides:
+      - goos: windows
+        format: zip
+
+changelog:
+  sort: asc
+  use: github
+  filters:
+    exclude:
+    - 'docs'
+    - '\.md'
+
+checksum:
+  name_template: checksums.txt
+  algorithm: sha256
+
+dockers:
+  - goos: linux
+    goarch: amd64
+    dockerfile: ./packages/docker/Dockerfile
+    image_templates:
+      - "boshprometheus/cf-exporter:latest"
+      - "boshprometheus/cf-exporter:{{ .Tag }}"
+      - "boshprometheus/cf-exporter:v{{ .Major }}"
+      - "boshprometheus/cf-exporter:v{{ .Major }}.{{ .Minor }}"
+    build_flag_templates:
+      - "--label=org.opencontainers.image.created={{ .Date }}"
+      - "--label=org.opencontainers.image.title={{ .ProjectName }}"
+      - "--label=org.opencontainers.image.revision={{ .FullCommit }}"
+      - "--label=org.opencontainers.image.version={{ .Version }}"
+      - "--label=org.opencontainers.image.source={{ .GitURL }}"
```

Comment:
need to adapt this part, Dokerfile is located in to root-dir and labels must match bosh_exporter repository, not cf_exporter